com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/DeleteServicesStep.java

```diff
@@ -93,17 +86,14 @@ private void getServicesGuids(List<String> servicesToDeleteNames, ExecutionWrapp
     }
 
     private void validateGuidMapping(List<String> servicesToDelete, Map<String, String> serviceGuids, ExecutionWrapper execution) {
-        servicesToDelete.stream()
+        List<String> servicesWithoutGuid = servicesToDelete.stream()
             .filter(name -> serviceGuids.get(name) == null)
-            .forEach(invalidEntry -> execution.getStepLogger()
-                .error(Messages.ERROR_MAPPING_SERVICE_NAME_TO_GUID, invalidEntry));
+            .collect(Collectors.toList());
 
-        Optional<String> firstServiceWithoutGuid = servicesToDelete.stream()
-            .filter(name -> serviceGuids.get(name) == null)
-            .findFirst();
+        String names = String.join(",", servicesWithoutGuid);

```

Comment:
Could you remove this?
And on line 96 to do: throw new SLException(Messages.ERROR_MAPPING_SERVICE_NAMES_TO_GUIDS, String.join(",", servicesWithoutGuid););